silkworm/db/buffer.cpp

```diff
@@ -228,7 +228,7 @@ void Buffer::insert_receipts(uint64_t block_number, const std::vector<Receipt>&
     }
 }
 
-void Buffer::insert_header(BlockHeader block_header) {
+void Buffer::insert_header(BlockHeader& block_header) {
```

Comment:
OK, but pls then pass by const ref and remove std::move in the body

Reply:
Sure, sorry for missing that